build.gradle

```diff
@@ -11,11 +11,11 @@ buildscript {
 allprojects {
 
     ext {
-        appName = 'Shattered Pixel Dungeon'
-        appPackageName = 'com.shatteredpixel.shatteredpixeldungeon'
+        appName = 'Windswept Pixel Dungeon'
+        appPackageName = 'com.windsweptpixel.windsweptpixeldungeon'
 
         appVersionCode = 532
```

Comment:
Nice... I think you must have read [recommended-changes](https://github.com/00-Evan/shattered-pixel-dungeon/blob/master/docs/recommended-changes.md) as I reset appVersionCode to 0 on my first run --- and as the doc indicates, this is problematic without a bunch of additional follow-up.